budgetportal/models.py

```diff
@@ -1607,29 +1608,42 @@ def get_preview_page(self, financial_year_id, phase_slug, government_slug, spher
             programme_ref,
             function_ref
         ]
-
         expenditure_results = openspending_api.aggregate(cuts=expenditure_cuts, drilldowns=expenditure_drilldowns)
 
+        # We do a separate call to always build focus area data from the main appropriation phase
+        focus_cuts = [
+            openspending_api.get_adjustment_kind_ref() + ':' + '"Total"',
+            openspending_api.get_financial_year_ref() + ':' + '{}'.format(
+                FinancialYear.start_from_year_slug(financial_year_id)
+            ),
+            openspending_api.get_phase_ref() + ':' + '"{}"'.format(EXPENDITURE_TIME_SERIES_PHASE_MAPPING["original"]),
+        ]
+        focus_drilldowns = [
+            department_ref,
+            geo_ref,
+            function_ref
+        ]
+
+        focus_results = openspending_api.aggregate(cuts=focus_cuts, drilldowns=focus_drilldowns)
+
+        # Remove Direct Charge against the National Revenue Fund programmes
```

Comment:
that's weird, I don't expect preview pages to exclude DRF any more - that's a change I made in the focus area pages branch.